### PR TITLE
made astral-portfolio-page a guided project

### PR DESCRIPTION
**Pull request by tamannasgh**
really really sorry, i messed up, so i closed the last pr.
this is my new pr with the same issue, kindly check this and let me know if now its okay.
Working on Astral Portfolio Project #33

### CODE REVIEW

File: astral-portfolio-page/step1/Instructions.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+in this step we'll be writing some html!
+
+# let's write the markup!
+
+## start with index.html
+
+    - take the icons from ![font awesome](https://fontawesome.com/v4/icons/) we will use font awesome 4.
+    - paste this cdn in your index.html <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (Pranavtechie) [inline astral-portfolio-page/step1/Instructions.md:8]: You can mention the icon names here as well.

File: astral-portfolio-page/step1/challenges.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: 'Writing html files in all pages!'
```
Comment (Pranavtechie) [inline astral-portfolio-page/step1/challenges.yml:1]: Avoid using `!` exclamation mark at the end of step names and heading, these as the step names will be displayed together in the steps side-bar

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
Instructions and challenges are pretty much the same, you can write more detailed instructions in the instructions, you can refer from `https://codedamn.com/project/multiverse-html5-photo-gallery`, this has actually given a lot of snippets. 

You don't need to give all the required steps, but a lot of snippets will be better

**Comment by Tamanna-S** [comment]
alright @Pranavtechie and thank you vary much.

File: astral-portfolio-page/step1/Instructions.md (hunk 1 above)
Comment (Tamanna-S) [reply astral-portfolio-page/step1/Instructions.md]: done

**Comment by Tamanna-S** [comment]
hey @Pranavtechie  sorry for the delay, may you please check now and lemme know if that's okay.

**Comment by Tamanna-S** [comment]
> hey @Pranavtechie sorry for the delay, may you please check now and lemme know if that's okay.

.

**Review by Pranavtechie — APPROVED** [review]

**Comment by Pranavtechie** [comment]
@tamannasgh Please share your email address here or reach out to me at [pranav@codedamn.com](mailto:pranav@codedamn.com)

**Comment by tamannasgh** [comment]
hey @Pranavtechie here it is:
ts.harma2882006@gmail.com

**Pull request merged**